### ISSUE TITLE
Hunter quest for taming doesn't give [Feed pet]

### ISSUE THREAD

**Issue by Hanzemon** [elysium-project/bug-tracker#521]
I am in my Dwarf hunter Thwaynn and I reached level 11, I completed the hunter quest to get to tame beasts and I got all the abilities except for [feed pet], I tried logging of and restarting the game but the ability is missing in the spell book.

My character is lvl 11 Dwarf Thwaynn in Nighthaven (PvP)

**Comment by Terranzin** [comment]
Have you done ALL quests in the chain? It´s after the part where you tame 3 different pets, eventually you´ll get a quest to go see a certain trainer, this one gives you the feed pet. Since you havent mentioned you actually got up to that part.

**Comment by Hanzemon** [comment]
Yeah I am sorry, this has never happened to me. I've always played other races in the horde and the quests just go fluidly like I didn't have to travel to complete the quest last time I played hunter in vanilla (last week), I was insecure and I started asking in world chat (bad mistake). People got kinda trolly and were telling me to delete my character and others told me to open a ticket and so... I already got my feed pet in Ironforge, anyways sorry for wasting your time and thank you for responding so rapidly. 

Peace.
